packages/lesswrong/components/linkPreview/FootnotePreview.tsx

```diff
@@ -319,44 +319,57 @@ function getFootnoteId(href: string, html: string): string|null {
  * Because of different versions of the CkEditor footnote plugin, imported
  * posts, etc, we have several different strategies for doing this:
  *  - Find the footnote reference, and use the data-footnote-index prop
- *  - Find the footnote in the post footer, and if it's inside an <ol> tag, use
+ *  - Find the footnote in the page footer, and if it's inside an <ol> tag, use
  *    its position within that tag's list of children
  */
 function getFootnoteIndex(href: string, html: string): string|null {
-  // Parse the footnote for its data-footnote-id, use that to find the footnote
-  // reference (which is a span that wraps around the part of the footnote that
-  // got hover-preview-ified), and take the data-footnote-index from that to
-  // display.
```

Comment:
Was this comment incorrect?

Reply:
I thought that the other comments covered everything